notebooks/official/bigquery_ml/bqml-online-prediction.ipynb

```diff
@@ -267,7 +267,7 @@
       },
```

Comment:
<p>Or, "with the new endpoint, you can now deploy your model."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
<p>Thanks, will change to "With the new endpoint, you can now deploy your model."</p>